api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -159,6 +165,44 @@ private void validateStopDatetime(Visit visit, Visit otherVisit, Errors errors)
 			errors.rejectValue("stopDatetime", "Visit.visitCannotContainAnotherVisitOfTheSamePatient", messageBuilder
 			        .toString());
 		}
-		
+
+	}
+
+	private void validateStartDateTimeToBirthdate(Visit visit, Errors errors) {
+
+		Patient patient = visit.getPatient();
+
+		if (patient != null) {
+			if (visit.getStartDatetime() != null && patient.getBirthdate() != null) {
+				if (!patient.getBirthdateEstimated() && visit.getStartDatetime().before(patient.getBirthdate())) {
+					errors.rejectValue("startDatetime", "Visit.startDateCannotBeBeforeBirthdate",
+							"This visit has a start date before birth date of the patient.");
```

Comment:
@wluyima Do I have to add this manually to these 20 files or is there a faster way to do that?